src/Airship/Internal/Decision.hs

```diff
@@ -559,12 +559,17 @@ l13 r@Resource{..} = do
         Nothing ->
             m16 r
 
-l07 r = do
+l07 r@Resource{..} = do
     trace "l07"
     req <- lift request
-    if requestMethod req == HTTP.methodPost
-        then m07 r
-        else lift $ halt HTTP.status404
+    if requestMethod req == HTTP.methodPost then m07 r
+    else do
+        m <- _contentType <$> get
+        when (isJust m) $ do
+            let (Just (ct, body)) = m
+            lift body >>= lift . putResponseBody
+            lift $ addResponseHeader ("Content-Type", renderHeader ct)
```

Comment:
I feel like I need to think about this some more. 
Initially it seems strange to be calling `contentTypesProvided` function because I think of that being used knowing the resource exists. It's expected to lookup `[("application/json", _),..]` but calling the function in the LHS suggests to me the resource definitely exists. 

I might be wrong but if that's the case then the behaviour would be subtly different to erlang webmachine and would surprise people.